ocdata/core/bulk.py

```diff
@@ -0,0 +1,287 @@
+import os
+import pickle
+
+import ase
+import numpy as np
+import pymatgen
+from pymatgen.core.surface import (
+    SlabGenerator,
+    get_symmetrically_distinct_miller_indices,
+)
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+
+from ocdata.configs.constants import COVALENT_MATERIALS_MPIDS, MAX_MILLER
+from ocdata.configs.paths import BULK_PKL_PATH, PRECOMPUTED_SLABS_DIR_PATH
+
+
+class Bulk:
```

Comment:
I feel like it makes sense to move slab generation to the surface class?

Reply:
Agreed. Especially if we don't need to be storing / using untiled slabs, we can club cutting + tiling together and move to the Surface / Slab class.

Reply:
Updated in the latest `slab.py`, see the `@classmethod`s: https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset/blob/refactor-v2/ocdata/core/slab.py#L75. Open to suggestions!